src/pkg/util/fs/proc/proc.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package proc
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// HasFilesystem returns whether kernel support filesystem or not
+func HasFilesystem(fs string) (bool, error) {
+	p, err := os.Open("/proc/filesystems")
+	if err != nil {
+		return false, fmt.Errorf("can't open /proc/filesystems: %s", err)
+	}
+	defer p.Close()
+
+	suffix := "\t" + fs
```

Comment:
Question, why do we split over tab + string and then use combination as suffix?

Reply:
/proc/filesystems content is either :
- `nodev\tsysfs`
- `\toverlay`
Be sure to match the exact file system between with `\t` and EOL